SignalRServiceBenchmarkPlugin/plugins/Plugin.Microsoft.Azure.SignalR.Benchmark/SignalRPlugin/MasterMethods/StopCollector.cs

```diff
@@ -13,15 +13,26 @@ public class StopCollector : IMasterMethod
     {
         public Task Do(IDictionary<string, object> stepParameters, IDictionary<string, object> pluginParameters, IList<IRpcClient> clients)
         {
-            Log.Information($"Stop collecting...");
+            System.Timers.Timer timer = null;
```

Comment:
@ccic   I put stopping and disposing timer in `finally` block

Reply:
That's ok for me.